src/remediations/fifi.js

```diff
@@ -759,8 +760,9 @@ async function dispatchRHCRequests ({executor, rhcWorkRequest}, playbook_run_id)
 
 async function dispatchRHCSatelliteRequests (rhcSatWorkRequest, playbook_run_id) {
     try {
+        const WorkRequest = _(rhcSatWorkRequest).map('rhcSatWorkRequest').value();
```

Comment:
Super small thing feel free to ignore, but this should probably be "workRequest" just to keep with naming conventions.